src/components/core/DpBulkEditHeader.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta } from '@storybook/addon-docs'
+import DpBulkEditHeader from './DpBulkEditHeader'
+
+<Meta
+    title="Components/BulkEditHeader"
+    component={DpBulkEditHeader}
+/>
+
+# BulkEditHeader
+
+The BulkEditHeader component allows users to perform bulk editing operations on a set of data or records.
+
+In a data management system, users often need to edit multiple records simultaneously.
+Instead of editing each record individually, a bulk edit functionality allows users to select multiple records
+and apply changes to specific fields in one go.
+
+## Custom usage
+
+```html
+<div>
+  <dp-bulk-edit-header
+      v-if="selectedItemsCount.length > 0"
+      :selected-items-count="selectedItemsCount.length"
+      selection-text="Elements"
+      @reset-selection="resetSelection"
+      v-bind="args">
+    <button
+        class="btn-icns u-m-0"
+        type="button"
+        @click.prevent="deleteSelection">
+      <i class="fa fa-times u-mr-0_125" />
+      Delete
+    </button>
+  </dp-bulk-edit-header>
+  <input
+      type="checkbox"
+      @click="toggleSelectAll"> Select all
+  <div
+      class="u-mt-0_5"
+      v-for="item in args.options"
+      :key="item.id">
+    <input
+        :id="item.id"
+        v-model="item.checked"
+        :checked="!item.checked === false"
+        type="checkbox"/>
+    {{ item.label }}
```

Comment:
see other comment (accessibility/label).

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/627fc1d2bf10ff7a9610d1976a2a4c74cc3003d9